### ISSUE TITLE
Changing max current should require user confirmation

### ISSUE THREAD

**Issue by glynhudson** [OpenEVSE/openevse_wifi_gui#12]
It should not be possible to change the max current accidentally. We should add a user confirmation when selecting a current change via the UI.

**Comment by jeremypoulter** [comment]
Will also move service level/max current to be advanced. Time as well should be moved to System settings

**Comment by chris1howell** [comment]
I think we need the ability to set the maximum capability of the station, a 32A station should bot be allowed to set an 80A pilot. In the past we have used the maximum but it requires recompiling the firmware for each value resulting in many versions of firmware. For security it would be great to set the station maximum as a write once. If a value is already in EEPROM changing the setting is denyed. Resetting would require erasing and re-flashing the firmware.

**Comment by jeremypoulter** [comment]
I agree with @chris1howell, but that really should be something that is in the OpenEVSE module rather that the WiFi module, @lincomatic what do you think? I think this is also important for the solar divert. currently we read the max value in and save in (in RAM) while we are doing the divert, which is a little risky. Would be a lot nicer to not touch the max value and just set the Pilot level and leave the max level well alone.

**Comment by chris1howell** [comment]
I posted an issue for the OpenEVSE firmware here:
https://github.com/lincomatic/open_evse/issues/118

**Comment by lincomatic** [comment]
Yes, I've been thinking about this lately, there needs to be a separate max current capacity setting saved somewhere... the one that's loaded from EEPROM at boot. let's talk about this more on the OpenEVSE firmware side.
